packages/tezos-ts-rpc/test/tezos-ts-rpc.spec.ts

```diff
@@ -106,6 +106,71 @@ describe('RpcClient test', () => {
     });
   });
 
+  describe('getDelegates', () => {
+    const sampleResponse = {
```

Comment:
Not important now, but it might be a good idea to name `sampleResponse` with proto number, or use an array (table based test styles).

Data shape might change in future protocol upgrades. 
This does not need to be addressed now, but worth thinking about for the future. May also be YAGNI :stuck_out_tongue_winking_eye:

Reply:
That's a good point maybe we should capture that into an issue?

Reply:
Good idea, created issue here: #27